README.md

```diff
@@ -242,11 +242,10 @@ const logger = new ConsoleLogger('SDK', LogLevel.INFO);
 
 // You will need AWS credentials configured before calling AWS or Amazon Chime APIs.
 const chime = new Chime({ region: 'us-east-1' });
-const endpoint = await chime.getMessagingSessionEndpoint().promise();
 
 const userArn = /* The userArn */;
 const sessionId = /* The sessionId */;
-const configuration = new MessagingSessionConfiguration(userArn, sessionId, endpoint.Endpoint.Url, chime, AWS);
+const configuration = new MessagingSessionConfiguration(userArn, sessionId, undefined, chime, AWS);
```

Comment:
Suggest adding a explanation here as well for why builders need to pass `undefined` for the `endpointUrl`.

Reply:
Not sure the cognitive overload on unimportant detail is needed for readme when already documented in doc.

Reply:
So the first place builders are going to look are at the README and seeing an `undefined` field is weird to start with, but agree they can go to `MessagingSessionConfiguration` documentation which as an extra hop. Can skip.